frontend/src/features/Vessel/layers/VesselAlertLayer.jsx

```diff
@@ -4,40 +4,30 @@ import VectorSource from 'ol/source/Vector'
 import Feature from 'ol/Feature'
 import Point from 'ol/geom/Point'
 import { Vector } from 'ol/layer'
-import { LayerProperties } from '../../../../domain/entities/layers/constants'
+import { LayerProperties } from '../../../domain/entities/layers/constants'
 
 import { getVesselAlertStyle } from './style'
 import {
   getVesselCompositeIdentifier,
   getVesselLastPositionVisibilityDates,
   Vessel,
   vesselIsShowed
-} from '../../../../domain/entities/vessel/vessel'
-import { useIsSuperUser } from '../../../../auth/hooks/useIsSuperUser'
-import { monitorfishMap } from '../../monitorfishMap'
+} from '../../../domain/entities/vessel/vessel'
+import { useIsSuperUser } from '../../../auth/hooks/useIsSuperUser'
+import { monitorfishMap } from '../../map/monitorfishMap'
+import { vesselsAdapter, vesselSelectors } from '../slice'
 
 const VesselAlertLayer = () => {
   const isSuperUser = useIsSuperUser()
 
-  const {
-    vessels,
-    hideNonSelectedVessels,
-    selectedVesselIdentity,
-    vesselsTracksShowed
-  } = useSelector(state => state.vessel)
-
-  const {
-    nonFilteredVesselsAreHidden
-  } = useSelector(state => state.filter)
-
-  const {
-    previewFilteredVesselsMode
-  } = useSelector(state => state.global)
-
-  const {
-    vesselsLastPositionVisibility,
-    hideVesselsAtPort
-  } = useSelector(state => state.map)
+  const hideNonSelectedVessels = useSelector(state => state.vessel.hideNonSelectedVessels)
+  const vesselsTracksShowed = useSelector(state => state.vessel.vesselsTracksShowed)
+  const selectedVesselIdentity = useSelector(state => state.vessel.selectedVesselIdentity)
+  const vessels = useSelector(state => vesselSelectors.selectAll(state.vessel.vessels))
+  const nonFilteredVesselsAreHidden = useSelector(state => state.filter.nonFilteredVesselsAreHidden)
+  const previewFilteredVesselsMode = useSelector(state => state.global.previewFilteredVesselsMode)
+  const hideVesselsAtPort = useSelector(state => state.map.hideVesselsAtPort)
+  const vesselsLastPositionVisibility = useSelector(state => state.map.vesselsLastPositionVisibility)
```

Comment:
```suggestion
  const {
    hideNonSelectedVessels,
    vesselsTracksShowed,
    // ...
  } = useMainAppSelector(state => ({
    hideNonSelectedVessels: state.vessel.hideNonSelectedVessels,
    vesselsTracksShowed: state.vessel.vesselsTracksShowed,
    // ...
  }))
```

\+ Même remarque pour les autres layers.